src/main/java/wolox/training/repositories/BookRepository.java

```diff
@@ -0,0 +1,8 @@
+package wolox.training.repositories;
+
+import org.springframework.data.repository.Repository;
+import wolox.training.models.Book;
+
+public interface BookRepository extends Repository<Book, Long> {
+    Book findBookByAuthor(String author);
```

Comment:
The naming template lacks the Class mention. So it should be findByAuthor(String author)